client/components/payment-activity/index.tsx

```diff
@@ -0,0 +1,261 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { useState } from 'react';
+import {
+	Card,
+	CardBody,
+	CardHeader,
+	Button,
+	Dropdown,
+	DropdownMenu,
+} from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import moment from 'moment';
+import { DateRange } from '@woocommerce/components';
+import interpolateComponents from '@automattic/interpolate-components';
+
+/**
+ * Internal dependencies.
+ */
+import EmptyStateAsset from 'assets/images/empty-activity-state.svg?asset';
+
+import './style.scss';
+
+const PaymentActivity: React.FC = () => {
+	const {
+		accountDefaultCurrency,
+		transactions_data: { tpv },
+	} = wcpaySettings;
+	const [ before, setBefore ] = useState( moment() );
+	const [ after, setAfter ] = useState(
+		moment().subtract( 7, 'days' ).startOf( 'day' )
```

Comment:
The idea is the **default** stats will be for the past week, hence using the 7 here. Once the user interacts with the filters, this value will change accordingly.

Reply:
Does it make sense to read this value always from the "compared to" filter? We can probably depend on 7 being the default value for that filter.

Reply:
I am not against having a default value here, just against the idea of hard-coding. In that way, this component seems to be doing more than its paygrade.

Reply:
I agree that we shouldn't hard-code things when it's not needed, but I don't think that depending on the "compared to" filter is the way to go.

I think the two should be separate or at least the other way around, "Compared to:" depending on the "From:" filter. Users will select a date from the "From:" filter first and expect to have the "Compared to:" automatically follow the time range.

However, I believe these will be more apparent when starting the "data connection" phase. So if I'm handing this over to another team, I prefer to keep it separate so they can work on their preferred logic afterward. Otherwise, I might be introducing an extra burden to understand my logic...

What do you think?

Reply:
Yep, that makes sense 👍 WDYT about putting a TODO note about it in code and keeping this thread open (unresolved)?

Reply:
Agree! Added a TODO and linked to this thread 2903a10c29ac6e1a9bb664dc58f35070220537db